drivers/RFM69/new/RFM69_new.cpp

```diff
@@ -239,9 +248,58 @@ LOCAL void RFM69_clearFIFO(void)
 LOCAL void RFM69_interruptHandler(void)
 {
 	// set flag
-	RFM69_irq = true;
+	//RFM69_irq = true;
+
+
+	if (RFM69_receiveCallback) {
+		// Start checking if RX-FIFO is not empty, as we might end up here from an interrupt
+		// for a message we've already read.
+		//while (RFM69_available()) {
+			RFM69_receiveCallback();		// Must call RF24_readMessage(), which will clear RX_DR IRQ !
```

Comment:
Update the comments, remove commented code

Reply:
These comments should be deleted in the latest version as I remember.